package/doc/sphinx/source/documentation_pages/selections.rst

```diff
@@ -54,6 +54,7 @@ packages with the help of :ref:`Selection exporters`.
         True
 
 
+    For further details , see :ref:`ordered-selections-label`
```

Comment:
There is an extra space before the comma here that can be removed. Could you also add a trailing period at the end of this line as well?

```suggestion
    For further details on ordered selections, see :ref:`ordered-selections-label`.
```

Reply:
done

Reply:
done